### PR TITLE
updated authorizer reference to document v2 apis

### PR DESCRIPTION
**Pull request by ogazitt**

### CODE REVIEW

File: docs/authorizer-guide/decisiontree.mdx [hunk 1]
```diff
@@ -6,20 +6,90 @@ description: Documentation for the decisiontree API
...
+
+```json
+{
+  "identityContext": {
+    "type": "IDENTITY_TYPE_SUB",
+    "identity": "<subject>"
+  },
+  "policyContext": {
+    "decisions": ["visible", "enabled"],
+    "id": "[policy-id]",
```
Comment (carabasdaniel) [inline docs/authorizer-guide/decisiontree.mdx]:
```suggestion
```

File: docs/authorizer-guide/policy-instance.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_label: Policy Instance
+title: Policy Instance
+description: The Policy Instance
+---
+
+# Policy Instance
+
+The `is`, `decisiontree`, and `query` APIs for the Aserto Hosted Authorizer take a Policy Instance argument from the calling application.
```
Comment (carabasdaniel) [inline docs/authorizer-guide/policy-instance.mdx]:
```suggestion
The `is`, `decisiontree`, `query`  and `compile` APIs for the Aserto Hosted Authorizer take a Policy Instance argument from the calling application.
```

File: docs/authorizer-guide/policy.mdx [hunk 3]
```diff
@@ -9,6 +9,85 @@ description: The Policy API
...
+- `policies`
+- `policies/{id}`
+
+Each of these APIs is a `GET` style API.
+
+## policies
+
+`GET .../api/v2/policies?field_mask=<field1>,<field2>`
+
+This will return a list of all the policies loaded in the Authorizer.
```
Comment (carabasdaniel) [inline docs/authorizer-guide/policy.mdx]:
```suggestion
This will return a list of all the policies loaded in the Authorizer runtime.
```

**Review by carabasdaniel — COMMENTED** [review]
Overall looks good, I left a couple of suggestions but I think we should probably add the `/v2/compile` for partial evaluations into the docs also.

**Comment by ogazitt** [comment]
> Overall looks good, I left a couple of suggestions but I think we should probably add the `/v2/compile` for partial evaluations into the docs also.

I agree we should, but that can be a different commit. This is phase 1 of N :)

**Review by carabasdaniel — APPROVED** [review]

**Pull request merged**